src/components/media/controls/commonControls.tsx

```diff
@@ -63,6 +67,14 @@ export function ProgressSlider(props: MediaControls) {
         className="rustic-progress-buffered"
         variant="determinate"
         value={bufferedPercent}
+        sx={{
+          backgroundColor: theme.palette.action.focus,
+          '& .MuiLinearProgress-bar': {
+            backgroundColor: isDarkTheme
+              ? theme.palette.secondary.light
```

Comment:
```suggestion
              ? theme.palette.primary.light
```

Reply:
This is with primary.light. I don't think there's enough contrast.

<img width="432" alt="Screenshot 2024-05-26 at 4 36 50 PM" src="https://github.com/rustic-ai/ui-components/assets/111031789/ec0588e6-0030-4333-94c5-5c1c627f7453">

Reply:
I think the designs might have been updated after the issue as this is on the Figma

<img width="503" alt="Screenshot 2024-05-26 at 4 46 01 PM" src="https://github.com/rustic-ai/ui-components/assets/111031789/40c57efc-d74d-44da-a363-e8637a955e20">